src/rocksdb/src/tuning.rs

```diff
@@ -276,15 +286,28 @@ impl FromStr for CompressionType {
     type Err = anyhow::Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
-            "zstd" => Ok(Self::Zstd),
-            "snappy" => Ok(Self::Snappy),
-            "lz4" => Ok(Self::Lz4),
-            "None" => Ok(Self::None),
-            other => Err(anyhow::anyhow!(
-                "{} is not a supported compression type",
-                other
-            )),
+        let s = UncasedStr::new(s).as_str();
```

Comment:
still need to remove this `as_str()`, it'll implement `PartialEq<&str>` and the if's will work

Reply:
Juggling too many open PRs; ty for catching that.